protocol-units/execution/opt-executor/src/executor/execution.rs

```diff
@@ -64,10 +65,9 @@ impl Executor {
 		})
 		.await??;
 
-		warn!("Block execution compute the following state: {:?}", state_compute);
-
+		info!("Block execution compute the following state: {:?}", state_compute);
 		let version = state_compute.version();
-		debug!("Block execution computed the following version: {:?}", version);
+		info!("Block execution computed the following version: {:?}", version);
```

Comment:
No need to have two separate log events, tags can be used to distinguish the values:

```rust
info!(?state_compute, %version, "block execution complete")
```

Reply:
Submitted another improvement of this in #720.